third_party/CMakeLists.txt

```diff
@@ -9,11 +9,10 @@ if (BUILD_INDIGO OR BUILD_INDIGO_UTILS)
     if (BUILD_STANDALONE)
         add_subdirectory(libpng)
         add_subdirectory(pixman)
-        if (EMSCRIPTEN)
-            add_subdirectory(cairo-wasm)
-        else()
-            add_subdirectory(cairo)
+        if (UNIX AND NOT APPLE)
+            add_subdirectory(freetype)
```

Comment:
We also need to add possibility to use system freetype on Linux (see cairo or libpng or libz as an example).

Reply:
Fixed.
In case of NOT EMSCRIPTEN system freetype and fontconfig libs will be used.